cmd/talosctl/cmd/talos/events.go

```diff
@@ -9,13 +9,20 @@ import (
 	"fmt"
 	"os"
 	"text/tabwriter"
+	"time"
 
 	"github.com/spf13/cobra"
 
 	"github.com/talos-systems/talos/api/machine"
 	"github.com/talos-systems/talos/pkg/client"
 )
 
+var eventsCmdFlags struct {
+	tailEvents   int32
+	tailDuration time.Duration
+	tailID       string
+}
```

Comment:
I like this. We need more of this. I hate that we have so many global variables.

Reply:
yep, I try to do that at least for new commands 

the worst thing is that sometimes such variable is shared across commands :confused: